rust/libceed/src/basis.rs

```diff
@@ -238,6 +238,84 @@ impl<'a> Basis<'a> {
         })
     }
 
+    pub fn create_Hdiv(
```

Comment:
There need to be corresponding functions and docstrings in the sibling file `lib.rs`

Reply:
Fixed in [9f5fbf2](https://github.com/CEED/libCEED/pull/1156/commits/9f5fbf2d1bf33955b5a7006814dfefab7b7903f2).